pydisdrometer/ParsivelNasaGVReader.py

```diff
@@ -31,7 +45,7 @@ def read_parsivel_nasa_gv(filename, campaign='ifloods'):
     recalculate it based upon a fall speed relationship.
     '''
 
-    reader = NASA_APU_reader(filename, campaign)
+    reader = NASA_APU_reader(filename, campaign, skip_header)
 
```

Comment:
I like the idea of leaving it in. Perhaps it should be an int to be the number of lines to skip?